### ISSUE TITLE
Version 2.3.1 crashes on macOS 10.13.6 (High Sierra)

### ISSUE THREAD

**Issue by jri** [Sappharad/mp3gainOSX#25]
On macOS 10.13.6 (High Sierra) MP3Gain Express 2.3.1 crashes as soon as file processing starts. The GUI freezes. Version 2.3.0 worked fine.

According to crash report symbol `____chkstk_darwin` is missing. Possibly this is because `aacgain` is build for macOS 10.15.

![image](https://user-images.githubusercontent.com/120337/70907215-97649500-2008-11ea-9126-4aaeccd96460.png)

**Comment by Sappharad** [comment]
Oops. I guess 2.3.2 is needed then. Will try to address this in the next 48 hours.

**Comment by Sappharad** [comment]
This is more complicated than I expected. I went ahead and compiled aacgain on an OS X 10.7 Virtual Machine to ensure that it's compatible with 10.7.

But it appears that XCode 11 cannot compile builds of apps that run on 10.7, even though I'm targeting it. I haven't changed my settings since last year's release and am still targeting 10.7 but the builds crash there and the error I get suggest it's a problem with apple's clang compiler that they fixed once before a few years ago.

I'm just going to build a new release in a previous version of XCode, but first I need to set up a working environment with an older macOS version. I'm having problems with that too.

In the meantime I have disabled 2.3.1 from "Check for Updates" so nobody will accidentally upgrade. I'm hoping that I can get something worked out tomorrow.

**Comment by Sappharad** [comment]
Version 2.3.2 brings back support for 10.7 and newer. Unfortunately, it seems I can't easily have everything at once.

Building on XCode 10.1 on under macOS 10.13 (then signing & notarizing with 10.15) allows the app to run on 10.7 all the way to 10.15 just fine. But Dark Mode is missing, because the app is built prior to it being added in Mojave. Building in XCode 10.3 on macOS 10.14 seems to break support for 10.7 even though I'm targeting it, due to new entitlements required for app notarization. I thought this worked for 2.3 last year, but I guess not or I couldn't figure out exactly what combinations of versions I had used.

I've released 2.3.2 without Dark Mode, because it's better than nothing.

When it's time for 2.4 and I drop support for 10.7 - 10.10, I'll try to remember to test on all of those versions and build in a manner that supports 10.11 and up with all functionality.
